assets/src/scripts/lib/ajax.js

```diff
@@ -39,3 +39,50 @@ export function get(url, resolveWith='response') {
         req.send();
     });
 }
+
+/**
+ * Executes an HTTP GET request.
+ * @param  {String} url         URL to request
+ * @param  {String} resolveWith XMLHttpRequest attribute to resolve promise
+ *                              with. e.g., "response" or "responseXML"
+ * @return {Promise}
+ */
+export function post(url, sampleData, resolveWith='response', contentType = '') {
```

Comment:
If possible it'd be nice to merge the `get` and `post` into a single function, since the logic is largely the same. You could do this by changing the signature of the function to:

```
export function request(url, {data, method='GET', resolveWith='response', contentType=''}) {
```

and possibly wrappers for `get` and `post` that call this function.

Reply:
looks good to me. will do.

Reply:
I made this refactor and tests run locally passed - looking into Travis issue.

Reply:
Looks like I had to pass in empty objects in test calls.